x-pack/legacy/plugins/siem/server/lib/ecs_fields/index.ts

```diff
@@ -284,6 +284,21 @@ export const systemFieldsMap: Readonly<Record<string, string>> = {
   'system.auth.ssh.method': 'system.auth.ssh.method',
 };
 
+export const signalFieldsMap: Readonly<Record<string, string>> = {
+  'signal.original_time': 'signal.original_time',
+  'signal.rule.id': 'signal.rule.id',
+  'signal.rule.saved_id': 'signal.rule.saved_id',
+  'signal.rule.timeline_id': 'signal.rule.timeline_id',
+  'signal.rule.timeline_title': 'signal.rule.timeline_title',
+  'signal.rule.output_index': 'signal.rule.output_index',
+  'signal.rule.from': 'signal.rule.from',
+  'signal.rule.index': 'signal.rule.index',
+  'signal.rule.language': 'signal.rule.language',
+  'signal.rule.query': 'signal.rule.query',
+  'signal.rule.to': 'signal.rule.to',
+  'signal.rule.filters': 'signal.rule.filters',
+};
```

Comment:
Don't know if you want all the missing keys but there are a few.

They would be:

```ts
signal.rule.rule_id
signal.rule.false_positives
signal.rule.max_signals
signal.rule.risk_score
signal.rule.description
signal.rule.name
signal.rule.immutable
signal.rule.references
signal.rule.severity
signal.rule.tags
signal.rule.threats
signal.rule.type
signal.rule.size
signal.rule.enabled
signal.rule.created_at
signal.rule.updated_at
signal.rule.created_by
signal.rule.updated_by
signal.rule.version
```

Just throwing them out there if you see any you wanted.

Reply:
I can add them, I just did not need them